### PR TITLE
fix the cache error which also occurred error on run-test

### PR DESCRIPTION
**Pull request by JisanAR03**
the change will fix the cache error from home page and also fix the run-test error too

### CODE REVIEW

**Comment by JisanAR03** [comment]
@DonnieBLT , @arkid15r ,, there is a cache related issue I have notice , for that reason the home page not work perfectly . and also showing "run test" error in every PR . now It's fixed . can you please review for merge this update

**Review by DonnieBLT — APPROVED** [review]
Awesome thank you!

File: website/views.py [hunk 1]
```diff
@@ -272,6 +272,11 @@ def _cache_middleware(request, *args, **kwargs):
     return _decorator
 
 
+@receiver(user_logged_out)
+def handle_user_logged_out(request, user, **kwargs):
+    cache.clear()
```
Comment (arkid15r) [inline website/views.py:277]: Why do you clear the entire cache upon this event?

Comment (JisanAR03) [reply website/views.py:277]: yes sir, should i just remove the cache_key?

Comment (arkid15r) [reply website/views.py:277]: I'm not sure what issue you fixed with that but it seem an overkill to me in any case as the **entire** cache is being invalidated upon any user signin/signup/logout event, right?

Comment (JisanAR03) [reply website/views.py:277]: yeah right sir , I will fix this up for sure

**Pull request merged**